src/main/java/com/griefcraft/lwc/LWC.java

```diff
@@ -1824,10 +1825,10 @@ private void registerCoreModules() {
             try {
                 registerModule(new Factions());
             } catch (NoClassDefFoundError e) {
-                this.log("Failed to hook into Factions!");
+                plugin.getLogger().warning("Failed to hook into Factions!");
                 this.log("Please make sure you are using an updated version of FactionsUUID.");
                 this.log("https://www.spigotmc.org/resources/factionsuuid.1035/");
-                e.printStackTrace();
+                plugin.getLogger().log(Level.FINE, "Unable to find necessary FactionsUUID classes:", e);
```

Comment:
Don't know if we even need to log this -- the presence of this warning already implies that we have a NoClassDefFoundError.

Reply:
I leave it in case someone installs FactionsUUID and got the messages anyway, in order for you (eventually) to be able to get more informations without having to do something complicated.
But you're right, it may not be needed to log this.